DNN Platform/Library/Common/Globals.cs

```diff
@@ -3620,101 +3707,143 @@ public static string UserProfilePicRelativeUrl(bool includeCdv)
+        /// </summary>
+        /// <param name="currentDirectory">The current directory.</param>
+        /// <param name="strExtensions">The extensions to filter for.</param>
+        /// <param name="noneSpecified">If true, adds 'None Specified' to the list.</param>
+        /// <returns>A list of file names.</returns>
+        [Obsolete("This method has been deprecated. Scheduled removal in v10.0.0.")]
+        public static ArrayList GetFileList(
+            DirectoryInfo currentDirectory,
+            [Optional, DefaultParameterValue("")] // ERROR: Optional parameters aren't supported in C#
+            string strExtensions,
+            [Optional, DefaultParameterValue(true)] // ERROR: Optional parameters aren't supported in C#
+            bool noneSpecified)
         {
             var arrFileList = new ArrayList();
             string strExtension = string.Empty;
 
-            if (NoneSpecified)
+            if (noneSpecified)
             {
                 arrFileList.Add(new FileItem(string.Empty, "<" + Localization.GetString("None_Specified") + ">"));
             }
 
-            string File = null;
-            string[] Files = Directory.GetFiles(CurrentDirectory.FullName);
-            foreach (string File_loopVariable in Files)
+            string file = null;
+            string[] files = Directory.GetFiles(currentDirectory.FullName);
+            foreach (string fileLoopVariable in files)
             {
-                File = File_loopVariable;
-                if (File.IndexOf(".") > -1)
+                file = fileLoopVariable;
+                if (file.IndexOf(".") > -1)
                 {
-                    strExtension = File.Substring(File.LastIndexOf(".") + 1);
+                    strExtension = file.Substring(file.LastIndexOf(".") + 1);
                 }
 
-                string FileName = File.Substring(CurrentDirectory.FullName.Length);
+                string fileName = file.Substring(currentDirectory.FullName.Length);
                 if (strExtensions.IndexOf(strExtension, StringComparison.InvariantCultureIgnoreCase) != -1 || string.IsNullOrEmpty(strExtensions))
                 {
-                    arrFileList.Add(new FileItem(FileName, FileName));
+                    arrFileList.Add(new FileItem(fileName, fileName));
                 }
             }
 
             return arrFileList;
         }
 
-        [Obsolete("This method has been deprecated. Replaced by GetSubFolderPath(ByVal strFileNamePath As String, ByVal portaId as Integer).. Scheduled removal in v11.0.0.")]
+        /// <summary>
+        /// Gets the subfolder path for a give filename path.
+        /// </summary>
+        /// <param name="strFileNamePath">The filename full path.</param>
+        /// <returns>The subfolder name.</returns>
+        [Obsolete("This method has been deprecated. Replaced by GetSubFolderPath(ByVal strFileNamePath As String, ByVal portaId as Integer). Scheduled removal in v10.0.0.")]

```

Comment:
Why did this change to v10 instead of v11?

Reply:
Addressed in latest commit, thanks.